lisa/tools/__init__.py

```diff
@@ -44,11 +52,17 @@
     "Mkfsxfs",
     "Modinfo",
     "Mount",
+    "Ntp",
+    "Ntpstat",
     "Ntttcp",
     "Nvmecli",
     "Reboot",
     "Sed",
     "Uname",
+    "Service",
+    "Systemctl",
```

Comment:
The SystemCtl and ServiceInternal doesn't need to be written here, because user doesn't need to use them directly.